ambf_ros_modules/examples/dvrk_control_socket_example/README.md

```diff
@@ -0,0 +1,105 @@
+# dVRK Control via Socket
```

Comment:
Not technically the dVRK, should I be referring to simulation as dVRK?

Reply:
Hello Akhil, thanks for this PR. I think this one is more relevant to the SRC. What do you think?

Reply:
What do you mean by SRC?